src/v/resource_mgmt/cpu_profiler.h

```diff
@@ -70,12 +70,15 @@ class cpu_profiler : public ss::peering_sharded_service<cpu_profiler> {
 
     // Collects `shard_results()` for each shard in a node and returns
     // them as a vector.
```

Comment:
Missing doc for the new `filter_before` option.

Reply:
Added docs for this option.